src/chunk.c

```diff
@@ -134,6 +134,8 @@ static Chunk *get_chunks_in_time_range(Hypertable *ht, int64 older_than, int64 n
 
 #define ASSERT_IS_NULL_OR_VALID_CHUNK(chunk) Assert(chunk == NULL || IS_VALID_CHUNK(chunk))
 
+#define CHUNK_STATUS_DEFAULT (int16) 0
```

Comment:
Why do you need the cast? A zero is a zero in all types. It is also prudent to wrap constants that are not literal constants into parentheses to avoid problems.
```suggestion
#define CHUNK_STATUS_DEFAULT ((int16) 0)
```